system-container/run-system-container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+export ANSIBLE_LOG_PATH=/var/log/ansible.log
+exec ansible-playbook -i /etc/ansible/hosts ${OPTS} ${PLAYBOOK_FILE}
```

Comment:
We've added this part before `commit 18632f48e7c0ebe53ff1ec5bd894b3b198f769f2` so the two Dockerfiles were different.

As you pointed out, the best solution would probably be to add system containers support to playbook2image.